### ISSUE TITLE
PQR molecule import is hard to use (no sorting of results)

### ISSUE THREAD

**Issue by drew-parsons** [OpenChemistry/avogadrolibs#485]
**Avogadro version:**
 - Avogadrolibs: 1.93.0+git20200509.e169315-1
 - Qt: 5.14.2

**Desktop version:**
 - OS: Linux (debian unstable)
 - Version 5.7.6
 - Compiler gcc

**Describe the bug**

Avogadro2 has a facility to import molecules from the Pitt Quantum Repository (PQR).  That's great!  Except it's barely usable.

Entering in, say, the name of an amino acid, returns some 500 alternative molecules.  Not alternative structures for the molecule requested, but alternative derivatives, second cousins, all of Rabbit's friends and relations.  Probably the molecule you requested is there somewhere. But where?  How to find it?  You've already asked for it by name, what more is to be done?

The problem does not seem to be intrinsic to the PQR system.  If I enter the same molecule at http://pqr.pitt.edu/, then 100s of alternatives are offered, but the obvious one directly matching the name is at the top of the list.

e.g. the second simplest amino acid, "alanine" (curiously there is only one structure found for "glycine")
Searching by Name, http://pqr.pitt.edu/ returns 548 molecules, starting with: 
*    L-(+)-Alanine
*    D-(-)-Alanine
*    Fludalanine
*    3-Oxo-L-Alanine
*    3-Oxo-D-Alanine
*    A-Methylalanine
    etc...

while Avogadro returns also 548 entries, but starting with
*    (4r,7r,10s,13r,15e,17r,19s)-7-[(2-bromo-1h-indol-3-yl)methyl]-4-(4-hydroxyphenyl)-8,10,13,15,17,19-hexamethyl-1-oxa-5,8,11-triazacyclononadec-15-ene-2,6,9,12-tetrone
*    l-seryl-(3r)-n-[(4r,7s)-4-{[(1s)-1-carboxyethyl]carbamoyl}-12,14-dihydroxy-11-methyl-6,10-dioxo-1,3,4,5,6,7,8,10-octahydro-9,2,5-benzoxathiazacyclododecin-7-yl]-3-hydroxy-l-prolinamide
*    n-{[(3s)-2-(l-tyrosyl)-1,2,3,4-tetrahydro-3-isoquinolinyl]carbonyl}-l-phenylalanyl-l-phenylalanine
*    (3s,9s,12s)-12-isopropyl-3-methyl-6-methylene-9-(3-phenylpropyl)-1-oxa-4,7,10-triazacyclododecane-2,5,8,11-tetrone
*   n-{(2r)-2-[2-(hydroxyamino)-2-oxoethyl]-4-methylpentanoyl}-3-methyl-l-valyl-n-(2-aminoethyl)-l-alaninamide
*    glycyl-l-prolylglycylglycine
*    (2s)-2-amino-3-{[(2e)-4-methoxy-4-oxo-2-butenoyl]amino}propanoyl l-norvalinate
*    (r)-n-[(3,4-dihydro-8-hydroxy-3-methyl-1-oxo-1h-2-benzopyran-7-yl)carboxyl]phenylalanine
*    reverse triiodothyronine
 *   d-mataven
    etc...

"alanine" isn't even a substring in those molecules!

Something is badly wrong with the sort order importing from PQR in Avogadro.

**To Reproduce**
1. Go to 'File/Import/Import From PQR'
2. Click on 'Search By' (leave search type set to 'name'. But searching as 'synonym' also has the same problem)
3. Enter "alanine" in 'Search By' box and click on Search button.
4. See error: returns 548 molecules in some ungodly unsorted order.

**Expected behavior**

Expect a simple alanine molecule to appear among the first entries (with or without H atoms, whether L- or D- structures). Preferably the physiologically relevant variant first (L-(+)-alanine).

**Screenshots**
Attached.
![avogadro-PQR-alanine-Screenshot](https://user-images.githubusercontent.com/26508288/88480293-d045ee80-cf87-11ea-94e8-6324999c06b6.png)

**Comment by ghutchis** [comment]
I'm going to close this because I'm not sure if PQR is coming back as a server. (I'll certainly try, but 🤷‍♂️ )

**Comment by drew-parsons** [comment]
Ironic, since "Pitt Invests Big in Quantum Science and Engineering"  https://quantum.pitt.edu/

**Comment by ghutchis** [comment]
Yeah, the problem is on the IT side sadly. I'll see what I can do, but I may need to find a separate host.
